### ISSUE TITLE
Agenda search: option to show only one stance of notes that have both schedule and deadline

### ISSUE THREAD

**Issue by drcassar** [orgzly/orgzly-android#465]
I want to create a search using the agenda query (ad) that shows only notes with a deadline value. My query is `d.ge.0d .it.done ad.7`
When I do this, notes that have a deadline and a schedule time will appear two times (one at the scheduled date and one at the deadline date).
Is it possible to control the search to show only one instance of these notes? In my case I only want to show the note at the deadline date.
Thank you for this awesowe app!!

- [x] I have searched for existing issues that may be the same as or related to mine.

**Comment by nevenz** [comment]
Yes, currently notes are located (`d.ge.0d .it.done`) and then both scheduled and deadline times used to put them in agenda (`ad.7`). Search query is not considered there.

A new agenda option could be added to specify which types of notes' timestamps should be used for agenda.

So perhaps something like:
* `at.deadline` (use only deadline time)
* `.at.scheduled` (use all types of timestamps except scheduled)
* `.at.plain` (all except plain timestamps - once supported)
